app/services/issue_notification.rb

```diff
@@ -0,0 +1,27 @@
+class IssueNotification < ApplicationService
+  def initialize(task:)
```

Comment:
since you have only one argument you can use positional arguments for this case and have like `IssueNotification.new(task)`